src/components/storagePools/storageVolumeCreateBody.jsx

```diff
@@ -62,18 +63,24 @@ const VolumeDetails = ({ idPrefix, size, unit, format, storagePoolCapacity, stor
         ];
     } else if (['dir', 'fs', 'netfs', 'gluster', 'vstorage'].indexOf(storagePoolType) > -1) {
         validVolumeFormats = ['qcow2', 'raw'];
+        existingValidVolumeFormats.push('iso');
     }
 
     if (validVolumeFormats) {
-        formatRow = (
-            <FormGroup fieldId={`${idPrefix}-fileformat`} label={_("Format")}>
-                <FormSelect id={`${idPrefix}-format`}
-                    onChange={(_event, value) => onValueChanged('format', value)}
-                    value={format}>
-                    { validVolumeFormats.map(format => <FormSelectOption value={format} key={format} label={format} />) }
-                </FormSelect>
-            </FormGroup>
-        );
+        if (!format)
+            console.error("VolumeDetails internal error: format is not set for storage pool type", storagePoolType);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1327-20231124-101157-c40ec8ba-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)